interfaces/builtin/hidraw_test.go

```diff
@@ -219,6 +254,14 @@ SUBSYSTEM=="hidraw", SUBSYSTEMS=="usb", ATTRS{idVendor}=="ffff", ATTRS{idProduct
 	c.Assert(spec.Snippets(), HasLen, 1)
 	snippet = spec.Snippets()[0]
 	c.Assert(snippet, Equals, expectedSnippet2)
+
+	expectedSnippet3 := `IMPORT{builtin}="usb_id"
+SUBSYSTEM=="hidraw", SUBSYSTEMS=="usb", ATTRS{idVendor}=="abcd", ATTRS{idProduct}=="1234", ENV{ID_USB_INTERFACE_NUM}=="00", TAG+="snap_client-snap_app-accessing-2-devices"`
```

Comment:
Why `00` and not just `0`?

Reply:
I just checked the ID_USB_INTERFACE_NUM is set to`00`, it must be '00' or it cannot match this attribute.(I have manually tried).